feature-dapp-impl/src/main/java/io/novafoundation/nova/feature_dapp_impl/presentation/browser/main/DAppBrowserFragment.kt

```diff
@@ -212,6 +251,7 @@ class DAppBrowserFragment : BaseFragment<DAppBrowserViewModel>(), OptionsBottomS
 
     private fun attachSession(session: BrowserTabSession) {
         clearProgress()
+        session.initialize(requireContext())
```

Comment:
We must initialize a session here to be sure that it happens in a thread with looper

Reply:
We can make factory method for `BrowserTabSession` to be suspend and switch to Main thread via `withContext` to ensure `BrowserTabSession` constructor runs in main thread

We should try to avoid late initialization as much as possible - delayed initialization makes it harder to reason about the class and introduces nothing but room for future bugs